src/smart-components/group/group-table-helpers.js

```diff
@@ -40,48 +42,144 @@ DefaultPlatformPopover.propTypes = {
+                      cells: [
+                        roleName,
+                        description,
+                        permissions,
+                        <Fragment key={`${uuid}-modified`}>
+                          <DateFormat date={modified} type={getDateFormat(modified)} />
+                        </Fragment>,
+                      ],
+                    };
+                  })}
                 >
-                  {name}
-                </AppLink>
+                  <TableHeader />
+                  <TableBody />
+                </Table>
               ) : (
-                name
-              )}
-              {(isPlatformDefault || isAdminDefault) && (
-                <DefaultPlatformPopover
-                  id={`default${isAdminDefault ? '-admin' : ''}-group-popover`}
-                  uuid={uuid}
-                  key={`${uuid}-popover`}
-                  bodyContent={intl.formatMessage(isAdminDefault ? messages.orgAdminInheritedRoles : messages.usersInheritedRoles)}
-                />
-              )}
-            </div>
-          </Fragment>,
-          roleCount,
-          principalCount,
-          <Fragment key={`${uuid}-modified`}>
-            <DateFormat date={modified} type={getDateFormat(modified)} />
-          </Fragment>,
+                <Text className="pf-u-mx-lg pf-u-my-sm">{intl.formatMessage(messages.noRolePermissions)}</Text>
+              ),
+          },
+        ],
+      },
+      {
+        uuid: `${uuid}-members`,
+        parent: 3 * 1,
+        compoundParent: 2,
+        fullwidth: true,
+        noPadding: true,
+        cells: [
+          {
+            props: { colSpan: 7, className: 'pf-m-no-padding' },
+            title:
+              principalCount > 0 ? (
+                <Table
+                  aria-label="Simple Table"
+                  ouiaId="members-in-group-nested-table"
+                  variant={TableVariant.compact}
+                  cells={[
+                    intl.formatMessage(messages.orgAdmin),
+                    intl.formatMessage(messages.username),
+                    intl.formatMessage(messages.email),
+                    intl.formatMessage(messages.firstName),
+                    intl.formatMessage(messages.lastName),
+                    intl.formatMessage(messages.status),
+                  ]}
+                  rows={principalCount.map((member) => ({
+                    cells: [member.orgAdmin, member.username, member.email, member.firstName, member.lastName, member.status],
+                  }))}
+                >
+                  <TableHeader />
+                  <TableBody />
+                </Table>
+              ) : (
+                <Text className="pf-u-mx-lg pf-u-my-sm">{intl.formatMessage(messages.noGroupMembers)}</Text>
+              ),
+          },
         ],
-        selected: Boolean(selectedRows && selectedRows.find((row) => row.uuid === uuid)),
       },
-    ],
+    ]),
     []
   );
 };
+// cells: [
```

Comment:
it looks like the empty initial array for reduce is in the wrong position

```suggestion
    ], [])
  );
};
```

Reply:
ah, thank you!